app/graphql/mutations/events/update_event.rb

```diff
@@ -1,15 +1,19 @@
 module Mutations
   module Events
-    class UpdateEvent < ::Mutations::BaseMutation
+    class UpdateEvent < Mutations::BaseMutation
       argument :id, ID, required: true
       argument :name, String, required: false
       argument :date, String, required: false
+      argument :week_number, String, required: false
```

Comment:
I believe these should probably be Integers. Will have to update a few other things if we make this change

Reply:
Actually, week_num shouldn't even be accepted as an arg in update event

Reply:
Updated. Luke and I took this out, but I guess it didn't save. Thanks for catching this